### PR TITLE
Add test cases to check that command exception is thrown when ab is empty

### PR DESCRIPTION
**Pull request by myc37**

### CODE REVIEW

File: src/test/java/seedu/address/logic/commands/ClearCommandTest.java [hunk 1]
```diff
@@ -22,7 +22,7 @@ public void execute_emptyAddressBook_failure() {
     }
 
     @Test
-    public void execute_nonEmptyAddressBook_success() {
+    public void execute_emptyAddressBook_throwCommandException() {
```
Comment (sheimoria) [inline src/test/java/seedu/address/logic/commands/ClearCommandTest.java:25]: In line 30, is the `assertCommandSuccess(new ClearCommand(), ...)` method call supposed to be changed to `assertCommandFailure(new ClearCommand(), model, String.format(Messages.MESSAGE_EMPTY_CLIENT_LIST, "clear")`?

**Review by sheimoria — COMMENTED** [review]
Need some clarification on the changes in `ClearCommandTest.java`. Other than that, LGTM 👍🏽

File: src/test/java/seedu/address/logic/commands/ClearCommandTest.java (hunk 1 above)
Comment (myc37) [reply src/test/java/seedu/address/logic/commands/ClearCommandTest.java]: Resolved, thanks for pointing it out. I was supposed to delete this test case and replace it with execute_nonEmptyAddresBook_success() but forgot to do so.

**Review by sheimoria — APPROVED** [review]
LGTM 👍🏽
